cli/pkg/kctrl/cmd/app/init/init.go

```diff
@@ -222,12 +222,16 @@ func (c CreateStep) createAppFromAppBuild() kcv1alpha1.App {
 func (c CreateStep) configureExportSection() {
 	fetchSource := c.build.GetObjectMeta().Annotations[FetchContentAnnotationKey]
 	exportSection := *c.build.GetExport()
+	// TODO not handling the case: In case of pkg init rerun with FetchFromLocalDirectory,
```

Comment:
Are we saying we are not handling these scenarios now and we would want to handle these in future? If so, can maybe we can reword it a little?

Reply:
Reworded